### ISSUE TITLE
Enterprise bot template needs package versions updated

### ISSUE THREAD

**Issue by JasonSowers** [Microsoft/botbuilder-dotnet#1089]
## Describe the bug
Enterprise template uses packages with the 4.0.5-19 version.  So users are not able to just upgrade via nuget package manager without editing the versions in the .csproj to remove the "-19".
```xml
    <PackageReference Include="Microsoft.Bot.Builder" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Builder.TemplateManager" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Connector" Version="4.0.5-19" />
    <PackageReference Include="Microsoft.Bot.Schema" Version="4.0.5-19" />
```
## To Reproduce
1. Create a new Enterprise Bot Template
2. Try to update the nuget packages
## Expected behavior
SHould be able to update packages.

[bug]

**Comment by lauren-mills** [comment]
@JasonSowers I have this updated in the myget feed and I'm just waiting for credentials to publish

**Comment by sgellock** [comment]
@lauren-mills we're releasing 4.1.5 today.  Can you make sure your updated template takes into account this new release before you publish?  Thanks

**Comment by ryanlengel** [comment]
Resolved in Microsoft/AI#240
